docs/openapi/components/schemas/common/TraceablePresentation.yml

```diff
@@ -0,0 +1,96 @@
+title: TraceablePresentation
+type: object
+description: A JSON-LD Presentation which establishes traceability linkage to a workflow instance and workflow type. 
+  - type: object
+    properties:
+      workflow:
+        $ref: "./Workflow.yml"
+        description: Establishes a traceable link from the presentation to a workflow.
+    required: 
+      - workflow
+example:
+  {
+    "@context":
+      [
+        "https://www.w3.org/2018/credentials/v1",
+        "https://w3id.org/security/suites/jws-2020/v1",
+        "https://w3id.org/traceability/v1"
+      ],
+    "id": "urn:uuid:83432751123654",
+    "type": 
+      [
+        "VerifiablePresentation",
+        "TraceablePresentation"
+      ],
+    "workflow": {
```

Comment:
This uses the word workflow, in our definition we have a path defined not a workflow, really need to settle on one.

also the instance field, is this supposed to be correlationID?
what is definition field?